pennylane/queuing.py

```diff
@@ -246,7 +250,130 @@ def _get_info(self, obj):
 
         return self._queue[obj]
 
+    append = _append
+    remove = _remove
+    update_info = _update_info
+    get_info = _get_info
+
     @property
     def queue(self):
         """Returns a list of objects in the annotated queue"""
         return list(self._queue.keys())
+
+
+def apply(op, context=QueuingContext):
+    """Apply an instantiated operator or measurement to a queuing context.
+
+    Args:
+        op (.Operator or .MeasurementProcess): the operator or measurement to apply/queue
+        context (.QueuingContext): The queuing context to queue the operator to.
+            Note that if no context is specified, the operator is
+            applied to the currently active queuing context.
+    Returns:
+        .Operator or .MeasurementProcess: the input operator is returned for convenience
+
+    **Example**
+
+    In PennyLane, **operations and measurements are 'queued' or applied to a QNode
```

Comment:
hmm, strictly speaking they are queued to a tape. Would "circuit" be a sufficiently vague term?

Reply:
Or could we say, they are added to a circuit when they are instantiated in a qfunc?

Reply:
```suggestion
    In PennyLane, **operations and measurements are 'queued' or added to a circuit
```